openapi-cli/src/main/resources/templates/utils.bal

```diff
@@ -184,12 +184,12 @@ isolated function getSerializedRecordArray(string parent, record {}[] value, str
 #
 # + value - Value to be encoded
 # + return - Encoded string
-isolated function getEncodedUri(string value) returns string {
-    string|error encoded = url:encode(value, "UTF8");
+isolated function getEncodedUri(anydata value) returns string {
+    string|error encoded = url:encode(value.toString(), "UTF8");
     if (encoded is string) {
         return encoded;
     } else {
-        return value;
+        return value.toString();
```

Comment:
I'm afraid that if we return the error string value to the path URL, won't it be broken 🤔 ? how about if we can return a string or error for our `getEncodedUri` method :) ? WDYT?

Reply:
@lnash94 here we are no doing `toString` to the error. We `toString` the original value not the error received after encoding.

Reply:
+1 for it :)